cmd/cat-main.go

```diff
@@ -105,6 +162,13 @@ func catOut(r io.Reader, size int64) *probe.Error {
 	var n int64
 	var e error
 
+	// In case of a user showing the object content in a terminal,
+	// avoid printing control and other bad characters to avoid
+	// terminal session corruption
+	if terminal.IsTerminal(int(os.Stdout.Fd())) {
```

Comment:
IMO we would need to introduce an option and work accordingly exactly like `cat -A` option

We shouldn't display control chars in hex automagically.